bench_state.go

```diff
@@ -86,6 +86,7 @@ func (s *benchmarkState) printErrors(out output) {
 		total += v
 	}
 	out.Printf("Total errors: %v\n", total)
+	out.Printf("Error rate: %.4f\n", float32(total)/float32(len(s.latencies)))
```

Comment:
nit: latencies only includes requests that succeeded, so it's possible that all requests failed in which case this will probably get a panic for dividing by 0. Can we put this in a `if len(s.latencies) > 0` block